mocks.config.js

```diff
@@ -6,7 +6,7 @@ const IntermediateAPIHandler = require('./mocks/handlers/IntermediateAPIHandler.
 
 module.exports = {
   // Log level. Can be one of silly, debug, verbose, info, warn or error
-  log: 'info',
+  log: 'debug',
```

Comment:
Ik ben gewoon even benieuwd of je deze bewust op debug heb gelaten? Want dan kan ik zoiets gaan aanhouden zodat niet elke keer hier een commit van bijkomt. Misschien is 'info' wel wat fijner zodat je ook nog het opstarten van de bff kan lezen. Maar allebei is prima hoor

Reply:
Info is beter dan denk ik. Ik start die servertjes apart op dus dan maakt het minder uit.